### PR TITLE
Add high availability for Redis instances section on redis-pmem example.

### PR DESCRIPTION
**Pull request by marcemq**

### CODE REVIEW

**Comment by okartau** [comment]
I think the added part looks good (but I havent tried it myself... should do it..).
A nitpick note concerning whole redis-pmem-oprator file:
The abbreviated name is used in different ways: K8S and K8s, I propose to unify.
I don't know which is more correct, K8s or k8s. Likely both so that the 1st one used in text and 2nd form as part of other names. But I think K8S is not correct (makes sense as it is never ending in uppercase).

**Comment by pohly** [comment]
Olev Kartau <notifications@github.com> writes:
> The abbreviated name is used in different ways: K8S and K8s, I propose
> to unify.  I don't know which is more correct, K8s or k8s. Likely both
> so that the 1st one used in text and 2nd form as part of other
> names. But I think K8S is not correct (makes sense as it is never
> ending in uppercase).

I would avoid both as much as possible. It's colloquial and might not be
understood by everyone.

**Comment by okartau** [comment]
> I would avoid both

I agree. Good policy is to avoid in text and spell fully
We have to use abbreviated form in hostnames, paths sometimes.

**Comment by marcemq** [comment]
Thank you @okartau and @pohly for your comments, I've updated the branch.

File: examples/redis-operator.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ This readme describes a complete example to deploy a Redis cluster through the [
 * [Docker registry](https://docs.docker.com/registry/deploying/).
 
 ## PMEM-CSI installation
-The steps below describe how to install PMEM-CSI within a K8S cluster that contains one master and three worker nodes. We use a specific version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced.
+The steps below describe how to install PMEM-CSI within a kubernetes cluster that contains one master and three worker nodes. We use a specific version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced.
```
Comment (pohly) [inline examples/redis-operator.md]: s/kubernetes/Kubernetes/

File: examples/redis-operator.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ The steps to install the Redis operator by Spotahome are listed below:
 
 ## Verify that the Redis instances are using the volumes provided
 First, match the volumes claim name used by each Redis instance to the current `pvc` which is provisioned by `pmem-csi-sc-ext4`. Both items must be properly bound.
-Next, check the block devices in the worker nodes of your K8S cluster and match the ones under the pmem `block` device, as shown below.
+Next, check the block devices in the worker nodes of your kubernetes cluster and match the ones under the pmem `block` device, as shown below.
```
Comment (pohly) [inline examples/redis-operator.md]: s/kubernetes/Kubernetes/

File: examples/redis-operator.md [hunk 3]
```diff
@@ -111,4 +111,53 @@ The steps to start playing around with Redis cluster *through sentinel instances
     $ kubectl exec -it rfs-redisfailover-pmem-7595857d4c-c69xg -- redis-cli -h 10.244.3.4 -p 6379 GET hello
     "world!"
     ```
-4. Feel free to play around with the Redis cluster.
\ No newline at end of file
+4. Feel free to play around with the Redis cluster.
+
+## High availability for Redis instances
+In order to ensure high availability for Redis instances, it's required to use `hardAntiAffinity: True` and `storageClassName: pmem-csi-sc-late-binding` under redis deployment specification, i.e.:
```
Comment (pohly) [inline examples/redis-operator.md]: s/redis/Redis/

File: examples/redis-operator.md [hunk 4]
```diff
@@ -111,4 +111,53 @@ The steps to start playing around with Redis cluster *through sentinel instances
...
+pmem-csi-node-xwk2l                          Running   pmem-csi-clear-govm-worker1   172.17.0.3
+redisoperator-688f6f4fcc-l6p4f               Running   pmem-csi-clear-govm-worker1   10.244.2.2
+rfr-redisfailover-pmem-lb-0                  Running   pmem-csi-clear-govm-worker2   10.244.1.3
+rfr-redisfailover-pmem-lb-1                  Running   pmem-csi-clear-govm-worker1   10.244.2.4
+rfr-redisfailover-pmem-lb-2                  Running   pmem-csi-clear-govm-worker3   10.244.3.4
+rfs-redisfailover-pmem-lb-5c455d7974-9twjz   Running   pmem-csi-clear-govm-worker1   10.244.2.3
+rfs-redisfailover-pmem-lb-5c455d7974-dscmx   Running   pmem-csi-clear-govm-worker2   10.244.1.2
+rfs-redisfailover-pmem-lb-5c455d7974-jtznn   Running   pmem-csi-clear-govm-worker3   10.244.3.3
+```
+As it is shown, each `rfr-redisfailover-pmem-lb-X` gets assigned to a different kubernetes cluster node `pmem-csi-clear-govm-workerX`, where `X` &isin; {1,2,3}.
```
Comment (pohly) [inline examples/redis-operator.md]: s/kubernetes/Kubernetes/

**Pull request merged**